### DISCUSSION TITLE
Github Android - Something went wrong, try again

### DISCUSSION THREAD

**Discussion by setchy** [community/community#34094] [Mobile] [answered]
I consistently get a `Something went wrong, try again` error when trying to view details of a Release for https://github.com/renovatebot/renovate.

This error is easy to reproduce by either:
* searching for the repository then going Releases >> selecting any release
* opening a Release Notification

App Version: 1.81.0
OS Version: Android 13

**Comment by Iron-Ham** [comment]
HI @setchy,

Thank you for reporting. I've gone ahead and opened up an internal issue for the Android team to take a peek at.

**Comment by setchy** [comment, marked as answer]
Looks like this issue has been resolved in a recent release

🎉
